cfme/tests/cli/test_appliance_console.py

```diff
@@ -67,6 +75,54 @@ def test_appliance_console_set_timezone(timezone, temp_appliance_preconfig_modsc
     temp_appliance_preconfig_modscope.appliance_console.timezone_check(timezone)
 
 
+def test_appliance_console_datetime(temp_appliance_preconfig_funcscope):
+    """Grab fresh appliance and set time and date through appliance_console and check result"""
+    app = temp_appliance_preconfig_funcscope
+    command_set = ('ap', '', '3', 'y', '2020-10-20', '09:58:00', 'y', '')
+    app.appliance_console.run_commands(command_set)
+
+    def date_changed():
+        return bool(app.ssh_client.run_command("date | grep 'Tue Oct 20 10:00'").success)
```

Comment:
Consider providing a specific date format to date so that you are not relying on its default behavior which could change.